portal-impl/src/com/liferay/portal/metadata/TikaRawMetadataProcessor.java

```diff
@@ -129,6 +148,16 @@ protected Metadata extractMetadata(
 		}
 	}
 
+	protected boolean isSupported(String mimeType) {
```

Comment:
@guilhermedcamacho Please inline this method as the only one usage is inside current class.